
string param_id
float32 param_value
uint8 param_type
---
bool paramset_ack
